request-handler/src/main/java/au/org/aodn/aws/wps/QueryStringParameterParser.java

```diff
@@ -97,4 +99,19 @@ else if (operation instanceof GetCapabilitiesOperation)
         //  TODO: exception?
         return null;
     }
+
+
+    private String getMapValueIgnoreCase(String key, Map<String, String> map)
+    {
+        if(key != null && map != null) {
+            if (map.get(key.toLowerCase()) != null) {
```

Comment:
Mixed case is also OK e.g. 'Request' - perhaps just loop through the keys and find one which matches when converted to lower case